core/domain/exp_domain.py

```diff
@@ -2872,6 +3932,17 @@ def _migrate_to_latest_yaml_version(
     def from_yaml(cls, exploration_id, yaml_content):
         """Creates and returns exploration from a YAML text string for YAML
         schema versions 10 and later.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+            yaml_content: str. The YAML representation of the exploration.
+
+        Returns:
+            Exploration. The corresponding exploration domain object.
+
+        Raises:
+            Exception: The initial schema version of exploration is less than
+            or equal to 9.
```

Comment:
Indent by 4, since this is a continuation line and there can be multiple exceptions.